types/src/lib.rs

```diff
@@ -8,6 +8,9 @@ extern crate alloc;
 /// JSON-RPC 2.0 specification related types.
 pub mod jsonrpc;
 
+/// JSON-RPC 2.0 specification related types v2.
+pub mod jsonrpc_v2;
```

Comment:
Could be just `v2` I think? External users will use `jsonrpsee_types` anyway right?